user_guide/common_features_channels.rst

```diff
@@ -3,13 +3,45 @@
 Channels
 ========
 
-Collection of comprehensive customer-related data is a cornerstone of customer relations management.
 
-To represent different sources of customer-related data, OroCRM uses records of the Channel entity ("channels").
+Today, successful businesses usually have more than one sales site. These may be different online stores, 
+business-to-business enterprises, or business-to-customer outlets. Moreover, different customer and sales-related 
+information can be received from various survey-campaigns or membership in clubs, funds, charity events, etc. Gaining a 
+full understanding of all the information you receive from each of these sources, along with all the individual customer 
+information you collect from various other sources, becomes crucial. With OroCRM's Channels you can do both with minimum 
+effort. 
 
-For example, any store, on-line site, social activity, etc. may be represented with its own channel.  
+In OroCRM, each Channel record represents one sources of customer-related data.
+Upon creation, you can choose what kind of customer-related information will be collected from each source (in technical 
+words "what entities will be assigned to this channel"). 
+
+For example, you have two channels of customer related data: your on-line store and your office.
+While they are both related to your customers, the information and customer relations process is quite different.
```

Comment:
Edit: For example, say you have two channels of customer related data: your online store and your office. While they are both related to your customers, the information and customer-relations process is quite different.